lightstep/instrumentation/runtime/builtin.go

```diff
@@ -114,7 +114,7 @@ func getTotalizedAttributeName(n string) string {
 	switch x[len(x)-1] {
 	case "cycles":
 		return "cycle"
-	case "usage":
+	case "usage", "time":
```

Comment:
are the changes in this file expected? they don't appear related to the change describe here